js/connection.js

```diff
@@ -328,7 +330,8 @@ weechat.factory('connection',
     var attemptReconnect = function (bufferId, timeout) {
         $log.info('Attempting to reconnect...');
         var d = connectionData;
-        connect(d[0], d[1], d[2], d[3], d[4], function() {
+        // won't work if totp is mandetory
```

Comment:
Slight spelling error here, correct spelling is 'mandatory'.

Reply:
ty